vg/draw/canvas.go

```diff
@@ -294,6 +332,34 @@ func (c Canvas) Crop(minx, miny, maxx, maxy vg.Length) Canvas {
 	}
 }
 
+// Tile returns new Canvases corresponding to the reciever canvas tiled
+// into the given number of rows and columns (row major, starting from the
+// top left corner) where t, b, l, r, h,
+// and w are the padding above, below, to the left, to the right, between rows,
+// and between columns of the tiles, respectively.
+func (c Canvas) Tile(rows, cols int, t, b, l, r, h, w vg.Length) [][]Canvas {
```

Comment:
I'd prefer something like this too.

Also, I think changing the parameter names to: top, bottom, left, right, padx and pady would improve the docs (remember that people are likely to interact more with code completion suggestions than the godoc day to day.